addons/mail/data/mail_data.xml

```diff
@@ -67,96 +67,65 @@
+    % elif ctx.get('button_unfollow'):
+    % if ctx.get('actions'):
+       |
+    % endif
+    <a href="${ctx['button_unfollow']['url']}"><strong>${ctx['button_unfollow']['title']}</strong></a>
+    % endif
 
-                % if ctx.get('button_access') or ctx.get('button_follow') or ctx.get('button_unfollow'):
-                % if ctx.get('actions'):
-                   |
-                % endif
-                <a href="${ctx['button_access']['url']}" style="text-decoration:none; color: #a24689;"><strong>${ctx['button_access']['title']}</strong></a>
-                % endif
-                </p>
-            </td>
-        </tr>
-    </tbody></table>
+    % if ctx.get('button_access') or ctx.get('button_follow') or ctx.get('button_unfollow'):
+    % if ctx.get('actions'):
+       |
+    % endif
+    <a href="${ctx['button_access']['url']}"><strong>${ctx['button_access']['title']}</strong></a>
+    % endif
+    </p>
 </div>
-<div summary="o_mail_notification" style="padding:0px; width:600px; margin:0 auto; background: #FFFFFF repeat top /100%;color:#777777">
-    <table cellspacing="0" cellpadding="0" style="vertical-align:top; padding:0px; border-collapse:collapse; background:inherit; color:inherit">
-        <tbody><tr>
-            <td valign="top" style="width:600px; padding:0px 10px 5px 5px;">
-                <div>
-                    <hr width="100%" style="background-color:rgb(204,204,204);border:medium none;clear:both;display:block;font-size:0px;min-height:1px;line-height:0;margin:0px auto;padding:0">
-                </div>
-            </td>
-        </tr></tbody>
-    </table>
-</div>
-<div style="padding:0px; width:600px; max-width:600px; margin:0 auto; background: #FFFFFF repeat top /100%; color:#777777">
-    <table cellspacing="0" cellpadding="0" style="width:600px ;text-align:justify; margin:0 auto; border-collapse:collapse; background:inherit; color:inherit">
-        <tbody><tr>
-            <td style="padding:0px 10px 5px 5px;font-size: 14px;">
-                ${object.body | safe}
-                % if ctx.get('tracking'):
-                    <ul>
-                    % for tracking in ctx['tracking']
-                        <li>${tracking[0]} : ${tracking[1]} -&gt; ${tracking[2]}</li>
-                    % endfor
-                    </ul>
-                % endif
-            </td>
-        </tr></tbody>
-    </table>
-</div>
-<div style="padding:0px; width:600px; max-width:600px; margin:0 auto; background: #FFFFFF repeat top /100%;color:#777777">
-    <table cellspacing="0" cellpadding="0" style="width:600px ;text-align:justify; margin:0 auto; border-collapse:collapse; background:inherit; color:inherit">
-        <tbody>
-            % if ctx.get('signature'):
-            <tr>
-                <td style="padding:0px 10px 5px 5px;font-size: 14px; text-align: left;">
-                    ${ctx['signature'] | safe}
-                </td>
-            </tr>
+<div>
```

Comment:
Same remark as before, do not use div but tables, tr, td and always fixed with / borders / ... .